inference-engine/src/gna_plugin/backend/am_intel_dnn.hpp

```diff
@@ -133,6 +133,34 @@ class AMIntelDNN {
                                             true);
     }
 
+#if GNA_LIB_VER == 2
```

Comment:
There is no better place, so I will leave this comment here.
gna_float_runtime.cpp file contains a reference implementation of all transformations. I guess it is used by FP32 mode.
Did you test FP32 mode?

Reply:
FP32 mode is probably not tested.
I would create separate ticket for implementation and testing of "reference CNN2D FP32 transformation" or we risk newer-ending story.
What do you think?

Reply:
@bsochack Could you comment/resolve this issue and possibly approve this PR?